<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookCmdAction</class>
 <widget class="QWidget" name="LookCmdAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Btn_OK">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
	
	
                            color:#ffffff;
                          background-color: rgb(0, 170, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }

  QPushButton:hover{
                            color:#ffffff;
                                background-color: rgb(0, 255, 0);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(0, 255, 0);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="shortcut">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_NG">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_LOOKDIR">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_LOOKDIR">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>外观保存路径</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
